anaconda_project/project_commands.py

```diff
@@ -408,6 +408,8 @@ def description(self):
             description = self._attributes.get('windows', None)
         if description is None:
             description = self._attributes.get('conda_app_entry', None)
+        if description is None:
+            description = getattr(self, 'command', None)
```

Comment:
I originally wrote "why not `self.command` " but now I'm looking through the class and don't see a command attribute; is somewhere else monkeypatching this in? Maybe there should be a real setter/getter?